.github/workflows/add-review-team.yml

```
@@ -0,0 +1,31 @@
+  name: 'Label PR on review request'
+
+  on:
+    issue_comment:
+      types: [created]
+
+  jobs:
+    label-pr:
+      name: Label PR
+      if: ${{ github.event.issue.pull_request }} && !contains(github.event.issue.labels.*.name, 'review-requested')
+      runs-on: ubuntu-latest
+      steps:
+        - name: team-label
+          uses: actions/github-script@v6
+          with:
+            script: |
+              const teams = ['@conda-forge/staged-recipes', '@conda-forge/help-python', '@conda-forge/help-python-c', '@conda-forge/help-r', '@conda-forge/help-java', '@conda-forge/help-nodejs', '@conda-forge/help-c-cpp', '@conda-forge/help-perl', '@conda-forge/help-julia', '@conda-forge/help-ruby'];
```

```suggestion
              const teams = [
                '@conda-forge/staged-recipes',
                '@conda-forge/help-c-cpp',
                '@conda-forge/help-cdts',
                '@conda-forge/help-go',
                '@conda-forge/help-java',
                '@conda-forge/help-julia',
                '@conda-forge/help-nodejs',
                '@conda-forge/help-perl',
                '@conda-forge/help-python',
                '@conda-forge/help-python-c',
                '@conda-forge/help-r',
                '@conda-forge/help-ruby'
              ];
```
Added some missing teams/languages. Note that there is presently no ruby language team. Also, not sure if these line breaks are valid yaml syntax, but it makes the list easier to read and edit.

---

Thanks! The teams don't need to correspond to the ones currently available as this gets the teams from the ping but no-one will probably ping the team if it's non-existent. Some of the teams are currently empty though so we probably also need to have an internal discussion about what to do with the teams but that's probably out of the scope of this PR.

RE formatting: The whole thing is actually a string that is JavaScript so the formatting needs to be JS compliant. I don't program in JS though so no idea...

---

I think it's good to label with the correct language even if there is no one on the team. It allows us to identify if we need to recruit new team members instead of dumping all the less common languages on "staged-recipes"